pkg/iac-providers/terraform/commons/load-dir.go

```diff
@@ -35,10 +35,6 @@ import (
 )
 
 var (
-	// ErrEmptyTFConfigDir error
-	ErrEmptyTFConfigDir = fmt.Errorf("directory has no terraform files")
```

Comment:
These errors were consciously created by the earliers maintainers of terrascan so that they can accurately test for erraneous test cases.

Reply:
This change was made because we do not want a generic error to be propagated back to the caller functions.